### PR TITLE
fix: improved collapsable examples

### PR DESCRIPTION
**Pull request by burakukula**
In the next PR we will provide a custom template for examples, it will be a bit darker.

### CODE REVIEW

File: packages/website/components/GlobalStyles.tsx [hunk 1]
```diff
@@ -28,8 +28,8 @@ export const GlobalStyles = () => {
         body {
           color: ${tokens.gray800};
           font-family: ${tokens.fontStackPrimary};
-          font-size: ${tokens.fontSizeL};
-          line-height: ${tokens.lineHeightL};
+          font-size: ${tokens.fontSizeM};
+          line-height: ${tokens.lineHeightM};
```
Comment (bgutsol) [inline packages/website/components/GlobalStyles.tsx:32]: Do we really want to have code in the code blocks smaller? For me, it's a bit inconvenient right now to read it.

File: packages/website/components/LiveEditor/ComponentSource.tsx [hunk 2]
```diff
@@ -118,51 +147,78 @@ export function ComponentSource({
...
+              {showSource && (
+                <Flex>
+                  <CopyButton
+                    tooltipProps={{ placement: 'top' }}
+                    className={styles.copyButton}
+                    value={children}
+                    size="small"
+                  />
+                  {isExampleFromFile && (
+                    <Flex marginLeft="spacing2Xs">
```
Comment (gui-santos) [inline packages/website/components/LiveEditor/ComponentSource.tsx:200]: we don't need this Flex I think

we can use `gap` in the Flex containing these buttons

File: packages/website/components/LiveEditor/StaticSource.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import Highlight, {
   defaultProps as HighlightDefaultProps,
 } from 'prism-react-renderer';
-import theme from 'prism-react-renderer/themes/github';
+import github from 'prism-react-renderer/themes/github';
```
Comment (gui-santos) [inline packages/website/components/LiveEditor/StaticSource.tsx:6]: Same comment as before

**Review by gui-santos — APPROVED** [review]
Wow! Looks really cool

I have a couple of suggestions but they are not blockers 🙌

**Comment by gui-santos** [comment]
@burakukula 

Screenshot 2022-02-09 at 10 12 06

**Review by bgutsol — APPROVED** [review]
Love the styles of hidden code and an 👁 button ⭐️

Maybe we can have them all collapsed by default since the part of the code is visible even in collapsed state, wdyt? I remember we had all collapsed before v4

File: packages/website/components/GlobalStyles.tsx (hunk 1 above)
Comment (burakukula) [reply packages/website/components/GlobalStyles.tsx:32]: Yeah, that was the requirement from design.

**Pull request merged**